app/code/Magento/InventoryConfigurableProduct/etc/events.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Event/etc/events.xsd">
```

Comment:
We removed `quantity_below_group` from the wizard
So we don't need to process this option on backend